app/assets/javascripts/discourse/app/lib/uploads.js

```diff
@@ -279,40 +279,32 @@ export function getUploadMarkdown(upload) {
   }
 }
 
-export function displayErrorForUpload(data, siteSettings) {
+export function displayErrorForUpload(data, fileName, siteSettings) {
```

Comment:
This is a breaking change - I noticed at least one plugin is using it:

https://github.com/paviliondev/discourse-custom-wizard/search?q=displayErrorForUpload

Could we add the new parameter on the end, and have it default to `data.files[0].name` if not present?

Reply:
Ah damn I only searched inside discourse-org and discourse organizations. Sure I will do that, makes more sense to put the new param on the end.

Reply:
Added a deprecation warning too